sdk/forms/src/main/java/com/klaviyo/forms/InAppFormsConfig.kt

```diff
@@ -0,0 +1,10 @@
+package com.klaviyo.forms
+
+/**
+ * Configuration for in app forms
+ *
+ * @param sessionTimeoutDuration (seconds) timeout for listening to new in app forms to display
```

Comment:
We gotta come up with a better definition for this. Belle used

> The duration in seconds for which a form session should be extended.

But I don't think that's super intuitive either. How about

> Duration (in seconds) of the period of user inactivity after which an app session is terminated.

Reply:
cc @belleklaviyo 
lets workshop this comment and then use consistent language on both platforms